src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -133,6 +133,9 @@
       <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3bBuiltProjectOutputGroupDependencies%3bGetCopyToOutputDirectoryItems%3bSatelliteDllsProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
       <IncludeOutputGroupsInVSIXLocalOnly>DebugSymbolsProjectOutputGroup%3b</IncludeOutputGroupsInVSIXLocalOnly>
     </ProjectReference>
+    <ProjectReference Include="..\IssueViz\IssueViz.csproj">
```

Comment:
What about IncludeOutputGroupsInVSIX and IncludeOutputGroupsInVSIXLocalOnly?

Reply:
I'm not sure what added those properties or whether they are required.
The reference to `Core` doesn't include them, so I decided to start simple. It may be that we'll need to add them later as we add dependencies to IssueViz to make sure those dependencies end up in the vsix.